orchestration/runner.go

```diff
@@ -172,20 +176,27 @@ func (r *Runner) runContainer(logger Logger, vd *variantData) error {
 	return nil
 }
 
-func listServices(servicesFile string) ([]string, error) {
+func listServices(servicesFile string) ([]commons.Service, error) {
```

Comment:
I meant the `services` file, using yam to store the info in it instead of joining and splitting strings.
Like we already do with meta file for example.